### ISSUE TITLE
[Feature Request]: Check out at future date

### ISSUE THREAD

**Issue by phil-flip** [grokability/snipe-it#11356]
### Is your feature request related to a problem? Please describe.

We would like to plan the checkouts for items, that rotate alot.

### Describe the solution you'd like

Being able to plan a checkout at a future date. Causing a warning message, that a planned checkout has been already scheduled for the timeframe.

### Describe alternatives you've considered

Using a Calendar, like Outlook, to manage this. But this gets cluttered really easily with many items.

**Comment by phil-flip** [comment]
This is related to https://github.com/snipe/snipe-it/issues/7247

**Comment by vk-chaertl** [comment]
This feature would be most helpful to us, too.

**Comment by phil-flip** [comment]
I don't have high hopes… But let's see, if It's going to get implemented

**Comment by vk-chaertl** [comment]
It's been suggested numerous times before in different words, but Snipe's silent assignment removal kinda makes me feel as if there's something deep in the backend that makes this really difficult.

Because I can't imagine this use case doesn't look good to anyone.

**Comment by zsmbrvr** [comment]
Yes please! It would save me from having to edit the database manually...

**Comment by vk-chaertl** [comment]
> It's been suggested numerous times before in different words [...]

Just for completness' sake, here's the three issues I've found that are basically the same:

Related to #1050 #5081 and  #4345 to some degree.

**Comment by clicit** [comment]
Any chance that this will be possible? Or is there any other way how to schedule checkouts in the future? We need to constantly move equipment to different locations, and therefore pre-planning is necessary.

**Comment by wildeep** [comment]
We supply equipment to 3 different performance venues. Ideally as part of the planning stage, we'd like to 'reserve' assets starting at a date in the future, and extending for an arbitrary number of days. The reservation could be for a person or for a location.

As the planning for another event starts, we'd like to get a warning if we're attempting to reserve an asset that has previously been reserved for another performance.

**Comment by ginuso630-hub** [comment]
Hi, is there any news regarding this request?
We urgently need to set future dates for checking out and checking in the assets.
We use Snipe-IT Version v8.3.1 - build 19577 (master).

**Comment by naleyjanhelge** [comment]
For me, a feature like this would be extremely valuable.

I run a non-profit organization, and we have recently started purchasing equipment that is meant to be loaned out to young people. A common scenario is that a youth knows in advance that they will need a specific asset on a certain future date — for example, a camera to film their band’s concert in four weeks.

Today, they can submit a request immediately, but the problem is that this often means the asset ends up being effectively “reserved” for the entire period leading up to the event. That significantly reduces overall utilization of the equipment, which is something we want to avoid.

What would be ideal is the ability to request an asset for a future date or date range, without blocking the asset until that actual period begins. If another user then tries to request the same asset for an overlapping period, they should receive a clear message that there is already an approved request for that time window. But only if it is an approved request blocking it.

At the same time, for administrators reviewing requests, it would be extremely helpful to get a warning or conflict indicator if there are overlapping or competing requests for the same asset. This would prevent situations where multiple requests are approved for the same asset during the same period by accident.

In our organization, most equipment has a low barrier for borrowing, but in certain cases — such as filming a concert — the user really needs to be confident that the equipment will be available on that specific date. A future-dated reservation model with conflict detection would allow us to balance high utilization with predictable access.

I hope this use case makes sense and helps illustrate why this kind of functionality would be very useful.

Thanks again!
